mlflow/artifacts/__init__.py

```diff
@@ -59,7 +63,9 @@ def download_artifacts(
 
     store = _get_store(store_uri=tracking_uri)
     artifact_uri = store.get_run(run_id).info.artifact_uri
-    artifact_repo = get_artifact_repository(artifact_uri)
+    artifact_repo = get_artifact_repository(
+        add_databricks_profile_info_to_artifact_uri(artifact_uri, tracking_uri)
+    )
```

Comment:
This makes a difference in this line:

https://github.com/mlflow/mlflow/blob/134f280c3c245879e923f540ee3b10c9ac55fa22/mlflow/store/artifact/databricks_artifact_repo.py#L134-L137

Reply:
I'm surprised this has never come up as an issue. Do users not use profile scoping?